frontend-new/index.html

```diff
@@ -13,7 +13,7 @@
     <meta name="application-name" content="Hangar | PaperMC" />
     <meta name="theme-color" content="#ffffff" />
   </head>
-  <body>
+  <body class="background-body text-[#262626] dark:text-[#E0E6f0]">
```

Comment:
we don't want to hardcode colors like that, instead we want to put them as variables to form a proper color palette